projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -173,8 +173,9 @@ public static function load_editor_scripts() {
 
 		$data = array(
 			'defaults' => array(
-				'to'      => wp_get_current_user()->user_email,
-				'subject' => '[' . get_bloginfo( 'name' ) . ']' . ( isset( $post ) ? ' ' . esc_html( $post->post_title ) : '' ),
+				'to'                    => wp_get_current_user()->user_email,
+				'subject'               => '[' . get_bloginfo( 'name' ) . ']' . ( isset( $post ) ? ' ' . esc_html( $post->post_title ) : '' ),
+				'sendEmailOnSubmission' => apply_filters( 'grunion_should_send_email', true, null ),
```

Comment:
Should we pass the post_id here instead of null?  I know in some cases it is not there yet. 

We should also add a comment where this filter is documented.

Reply:
There is no post_id to add here, as this id is actually the `feedback` entity's id when is created after a form submission. It seems to just allow the possiblity of some internal usage of consumers of the filter (maybe for logging etc..).

You're right about the docs and it could create some problems to consumers like 170538-ghe-Automattic/wpcom.

I don't think it was a good idea to add this param to the filter in the first place, but here we are :)